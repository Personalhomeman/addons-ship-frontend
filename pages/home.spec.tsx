import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Home from './home';

it('renders correctly', () => {
  const tree = toJSON(shallow(<Home />));
  expect(tree).toMatchSnapshot();
});
