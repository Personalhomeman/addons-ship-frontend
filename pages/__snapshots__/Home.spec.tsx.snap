// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <_default>
    <_default
      iconUrl="/static/icon-flutter.svg"
    >
      My app
    </_default>
  </_default>
  <_default>
    Bitrise Ship Add-on frontend
  </_default>
  <Counter />
</Fragment>
`;
